includes/class-wc-payments-order-success-page.php

```diff
@@ -33,6 +35,89 @@ public function register_payment_method_override() {
 		add_filter( 'woocommerce_order_get_payment_method_title', [ $this, 'show_woocommerce_payments_payment_method_name' ], 10, 2 );
 	}
 
+	/**
+	 * Maybe render the payment instructions for Multibanco payment method.
+	 *
+	 * @param int $order_id The order ID.
+	 */
+	public function maybe_render_multibanco_payment_instructions( $order_id ) {
+		$order = wc_get_order( $order_id );
+
+		if ( ! $order || $order->get_payment_method() !== 'woocommerce_payments_' . Payment_Method::MULTIBANCO || 'on-hold' !== $order->get_status() ) {
+			return;
+		}
+
+		$order_service         = WC_Payments::get_order_service();
+		$multibanco_info       = $order_service->get_multibanco_info_from_order( $order );
+		$unix_expiry           = $multibanco_info['expiry'];
+		$expiry_date           = date_i18n( get_option( 'date_format' ) . ' ' . get_option( 'time_format' ), $unix_expiry );
```

Comment:
I think you could use `wc_date_format` and `wc_time_format` instead of getting the options directly, as apparently they could be empty.